.gitignore

```diff
@@ -20,3 +20,11 @@ contrib/build/*/tmp/*
 .pkg
 .kube
 .var
+charts/catalog/apiserver-key.pem
+charts/catalog/apiserver.csr
+charts/catalog/apiserver.pem
```

Comment:
I'm not sure, I know this works at the top level chart directory though, and I'd rather make progress this way for now